assets/control-plane/prometheus-rule.yaml

```diff
@@ -31,7 +31,8 @@ spec:
       expr: |
         sum by (namespace, pod, cluster) (
           max by(namespace, pod, cluster) (
-            kube_pod_status_phase{namespace=~"(openshift-.*|kube-.*|default)",job="kube-state-metrics", phase=~"Pending|Unknown"}
+            kube_pod_status_phase{job="kube-state-metrics", phase=~"Pending|Unknown"}
```

Comment:
This seems to be dropping (accidentally?) the `namespace=~"(openshift-.*|kube-.*|default)"` filtering.

Reply:
thank you, it is fixed now :)